eng/validation/remove-oldbranches.ps1

```diff
@@ -0,0 +1,98 @@
+Param(
+  [Parameter(Mandatory=$true)][string] $azdoOrg, 
+  [Parameter(Mandatory=$true)][string] $azdoProject,
+  [Parameter(Mandatory=$true)][string] $azdoToken,
+  [Parameter(Mandatory=$true)][string] $githubUser,
+  [Parameter(Mandatory=$true)][string] $githubPAT,
+  [Parameter(Mandatory=$true)][string] $githubOrg,
+  [Parameter(Mandatory=$true)][string] $githubRepoName,
+  [Parameter(Mandatory=$true)][string] $barToken, 
+  [string] $azdoRepoName
+)
+
+set-strictmode -version 2.0
+$ErrorActionPreference = 'Stop'
+
+. $PSScriptRoot\..\common\tools.ps1
+. $PSScriptRoot\..\common\darc-init.ps1
+
+$global:arcadeSdkPackageName = 'Microsoft.DotNet.Arcade.Sdk'
+$global:arcadeSdkVersion = $GlobalJson.'msbuild-sdks'.$global:arcadeSdkPackageName
+$global:azdoOrg = $azdoOrg
+$global:azdoProject = $azdoProject
+$global:azdoToken = $azdoToken
+$global:githubUser = $githubUser
+$global:githubPAT = $githubPAT
+$global:githubOrg = $githubOrg
+$global:githubRepoName = $githubRepoName
+$global:barToken = $barToken
+$global:azdoRepoName = if (-not $azdoRepoName) { "" } else { $azdoRepoName }
+
+# Get a temporary directory for a test root. Use the agent work folder if running under azdo, use the temp path if not.
+$testRootBase = if ($env:AGENT_WORKFOLDER) { $env:AGENT_WORKFOLDER } else { $([System.IO.Path]::GetTempPath()) }
+$testRoot = Join-Path -Path $testRootBase -ChildPath $([System.IO.Path]::GetRandomFileName())
+New-Item -Path $testRoot -ItemType Directory | Out-Null
+
+function GitHub-Clone($repoName) 
```

Comment:
Looks like there are some duplicated functions in this script and in `build-arcadewithrepo.ps1` - e.g., GitHub-Clone, Git-Command. I think it would be best to  extract them and put in a shared/common file.

Reply:
Removed this code.